eogrow/pipelines/rasterize.py

```diff
@@ -79,44 +43,79 @@ class Schema(Pipeline.Schema):
             description="The storage manager key pointing to the input folder for the rasterization pipeline."
         )
         output_folder_key: str = Field(
-            description="The storage manager key pointing to the output folder for the rasterization pipeline pipeline."
+            description="The storage manager key pointing to the output folder for the rasterization pipeline."
         )
         vector_input: Union[Feature, str] = Field(
```

Comment:
I left it like it was just to avoid too much change, but I would prefer to have `vector_feature` or `input_feature` and `filename` as separate and optional parameters.

> perhaps with the introduction of vector importing there is no longer a need for that

this is also a good point, but does that mean that the user should run a custom pipeline just for importing the vector data? from this point of view, the pipelines then become `Load + X + Save` pipelines. Not sure how to see the bigger picture here.

Reply:
maybe we should split it into two pipelines then. But yeah, a future MR perhaps